packages/features/bookings/lib/handleNewBooking/test/reschedule.test.ts

```diff
@@ -889,6 +891,203 @@ describe("handleNewBooking", () => {
           timeout
         );
 
+        test(
```

Comment:
This is inside 'User Event Type' describe. There is a 'Team Event Type' as well. We should move it below to that describe

Also, I believe the issue is with a regular User event as well if requires-confirmation is enabled, so we can have a test for User Event as well at this place.
<img width="607" alt="Screenshot 2024-05-02 at 1 53 24 PM" src="https://github.com/calcom/cal.com/assets/1780212/89d31585-fc62-46d3-8aab-b6f65bf9b5f9">